python/servo/command_base.py

```diff
@@ -868,17 +859,13 @@ def build_like_command_arguments(decorated_function):
             decorated_function = decorator(decorated_function)
         return decorated_function
 
-    def pick_target_triple(self, target, android, magicleap):
+    def pick_target_triple(self, target, android):
         if android is None:
             android = self.config["build"]["android"]
         if target and android:
             assert self.handle_android_target(target)
         if android and not target:
             target = self.config["android"]["target"]
-        if magicleap and not target:
-            target = "aarch64-linux-android"
-        if target and not android and not magicleap:
-            android = self.handle_android_target(target)
```

Comment:
These lines should stay; we should just remove the `and not magicleap` condition.